threadpool/core/src/main/java/cn/hippo4j/core/enable/BeforeCheckConfiguration.java

```diff
@@ -144,6 +144,33 @@ public BeforeCheckConfiguration.BeforeCheck dynamicThreadPoolBeforeCheckBean(@Au
                         }
                     }
 
+                    Map<String, String> polaris = properties.getPolaris();
+                    if (MapUtil.isNotEmpty(polaris)) {
+                        String namespace = polaris.get("namespace");
+                        if (StringUtil.isBlank(namespace)) {
+                            throw new ConfigEmptyException(
+                                    "Web server failed to start. The dynamic thread pool polaris namespace is empty.",
+                                    "Please check whether the [spring.dynamic.thread-pool.polaris.namespace] configuration is empty or an empty string.");
+                        }
+                        String fileGroup = polaris.get("file.group");
```

Comment:
Duplicate code maybe we can encapsulate in a method to make the code more concise.